docker/docker-compose.override.yml

```diff
@@ -1,25 +1,14 @@
-version: '3.2'
+version: '3.4'
 services:
   
   eth-node:
     restart: always
     build:
       context: ./development/eth-node
+    command: [ "yarn", "start", "--hostname", "0.0.0.0"]
```

Comment:
Maybe add the mnemonic so it always has the same accounts with some eth?

Reply:
yep, added in the hardhat config file already